Utils.lua

```diff
@@ -26,26 +26,54 @@ local function capitalize(string)
     return supper(ssub(string, 1,1)) .. slower(ssub(string, 2))
 end
 
-local classOrdered = {
-    "Death Knight", "Druid", "Hunter", "Mage", "Priest", "Rogue", "Shaman", "Paladin", "Warlock", "Warrior"
```

Comment:
Removing ordering (here it is alphabetical) results in ordering based on class number which doesnt look that nice:
![image](https://user-images.githubusercontent.com/43088589/198282732-2f53b424-02f5-446b-bcdc-c2f1079725ee.png)